### DISCUSSION TITLE
Welcome to tailwnd-nextjs-comming-soon Discussions!

### DISCUSSION THREAD

**Discussion by baikaresandip** [baikaresandip/tailwnd-nextjs-comming-soon#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by arts009009009** [comment]
Hey everyone 👋  
Glad to see this project starting up! I’m into Next.js myself, so this repo instantly caught my eye.  
Really curious to see how the “Coming Soon” template evolves — maybe adding more customization options or integrations down the line.  

Excited to be part of the discussions here 💪  
If you’d like to check out my own work, here’s my repo: https://github.com/arts009009009/Elite-Tech-Shop  
And here’s my profile if you’re curious about what else I’m building 😄: https://github.com/arts009009009
